troposphere/static/js/components/projects/detail/resources/ProjectDetails.react.js

```diff
@@ -101,10 +101,18 @@ define(function (require) {
     },
 
     onMoveSelectedResources: function () {
-      modals.ProjectModals.moveResources(
-        this.state.selectedResources,
-        this.props.project
-      );
+        var match = false;
+        this.state.selectedResources.forEach(function(sel){
+            if (stores.VolumeStore.getAttachedResources().indexOf(sel.get('uuid')) !== -1) {
+                match = true;
+            }
+        });
+        if (match) modals.ProjectModals.cantMoveAttached();
+        else
+        modals.ProjectModals.moveResources(
+            this.state.selectedResources,
+            this.props.project
+        );
```

Comment:
Please use curly braces around `if ... else`. 

I'm okay with something like: 
```javascript 
if (!this.models) return this.fetchModels();
```

But once you've got an `if ... else` block, it's better to surround the statements within curly braces.